tests/chainer_tests/serializers_tests/test_npz.py

```diff
@@ -269,17 +269,27 @@ def tearDown(self):
         if hasattr(self, 'temp_file_path'):
             os.remove(self.temp_file_path)
 
-    def test_load(self):
+    def test_load_with_strict(self):
         obj = mock.MagicMock()
         npz.load_npz(self.temp_file_path, obj)
 
         self.assertEqual(obj.serialize.call_count, 1)
         (serializer,), _ = obj.serialize.call_args
         self.assertIsInstance(serializer, npz.NpzDeserializer)
```

Comment:
Thank you for pointing out it. I added one line to check `serializer.strict` is True.